keras_cv/metrics/f_scores.py

```diff
@@ -0,0 +1,356 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+import tensorflow.keras as keras
+
+
+class FBetaScore(keras.metrics.Metric):
```

Comment:
Is there anything computer vision specific about this?  Could this also be used in NLP classification?  Perhaps this is better as a core Keras feature?

Reply:
What Is 2022 opinion of the team?

https://github.com/keras-team/keras/issues/5794

Reply:
Looks like Precision was reintroduced: https://www.tensorflow.org/api_docs/python/tf/keras/metrics/Precision

(wasn’t involved in anything regard this metric so perhaps I am incorrect)

Reply:
Also recall but you need to check internally with you team.

Reply:
Both Precision and Recall are in Keras now. We should add F Score